dev_guide/builds.adoc

```diff
@@ -223,6 +256,29 @@ cached layers and rerun all steps of the *_Dockerfile_*:
 ----
 ====
 
+[[force-pull]]
+
+=== Force Pull
+
+By default, if the builder Docker image specified in the  build confirguration is available in OpenShift's local,
+internal registry, that image will be used.  However, to override the local image and refresh it from the Docker image
+repository with a newer version, create a `*buildConfig*` with the "forcePull" flag set to true:
+
+====
+
+[source,json]
+----
+{
+  "strategy": {
+    "type": "Docker",
+    "dockerStrategy": {
+      "forcePull": true
```

Comment:
Yeah, did not add one here since one was not added for the NoCache one.  I'll switch to make consistent with s2i.